### PR TITLE
ACM-4170 fix not operator for cluster property

### PR DESCRIPTION
**Pull request by SherinV**
### Related Issue

https://issues.redhat.com/browse/ACM-4170

### Description of changes
- Update query when it includes the `cluster` property.

### CODE REVIEW

File: pkg/resolver/searchHelper.go [hunk 1]
```diff
@@ -55,31 +55,43 @@ func getOperator(values []string) map[string][]string {
...
 	case "!=":
-		exps = append(exps, goqu.L(`"data"->>?`, prop).Neq(values))
+		klog.Info("!=")
+		exps = append(exps, goqu.L(`?`, lhsExp).Neq(values))
+		klog.Info("!=", exps)
 
 	case "!":
-		exps = append(exps, goqu.L(`"data"->>?`, prop).NotIn(values))
+		exps = append(exps, goqu.L(`?`, lhsExp).NotIn(values))
+		klog.Info("!", exps)
```
Comment (jlpadilla) [inline pkg/resolver/searchHelper.go:83]: Please update these debug logs.

Comment (SherinV) [reply pkg/resolver/searchHelper.go]: Done. Thanks.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/cherry-pick release-2.7

**Pull request merged**